### ISSUE TITLE
Allow backslash commands

### ISSUE THREAD

**Issue by phiresky** [darold/pgFormatter#174]
Currently, the following

```sql
\include_relative ./main.sql
\include_relative ./a/b.sql
\include_relative ./c/d.sql
```

gets formatted to

```sql
include_relative./ main.sql include_relative./ a / b.sql include_relative./ c / d.sql
```

Would be great if all psql client commands could be preserved (like `\include_relative`, `\set` etc)

**Comment by darold** [comment]
Yes I have found several other problems relative to backslash with psql command yesterday, I am working on it.

**Comment by darold** [comment]
Fixed in commit e2b1fa0.
